metadata_backend/helpers/schemas/workflow.json

```diff
@@ -46,7 +46,8 @@
                             "description": "The other of the schema is the order it will render in the front-end",
```

Comment:
Noticed this sentence. Is it supposed to say `The order of the schema...`?

Reply:
It is 😅